### ISSUE TITLE
Ace by DAISY reports errors for links

### ISSUE THREAD

**Issue by rtonsing** [gutenbergtools/ebookmaker#316]
Mainly for header/footer links to PG. Seems contradictory to me, "insufficient color contrast" vs. "Links must be distinguishable without relying on color".

```
The link has insufficient color contrast of 2.23:1 with the surrounding text. (Minimum contrast is 3:1, link text: #0000ee, surrounding text: #000000)
The link has no styling (such as underline) to distinguish it from the surrounding text
Ensure links are distinguished from surrounding text in a way that does not rely on color
Links must be distinguishable without relying on color
```

[pg78913.zip](https://github.com/user-attachments/files/29206676/pg78913.zip)

**Comment by eshellman** [comment]
I thought I fixed this in https://github.com/gutenbergtools/ebookmaker/commit/b3709cafd987ddcdd31cddc2b1ac0b1af4f643ea
